src/System.Net.Sockets/src/System/Net/Sockets/Socket.Windows.cs

```diff
@@ -241,7 +241,6 @@ private IAsyncResult BeginSendFileInternal(string fileName, byte[] preBuffer, by
             FileStream fileStream = OpenFile(fileName);
 
             TransmitFileAsyncResult asyncResult = new TransmitFileAsyncResult(this, state, callback);
-            asyncResult.StartPostingAsyncOp(false);
```

Comment:
From the old code, these appear to ensure that the correct `Context` and `Identity` are preserved. 
Why is it safe to remove them now? (Is .Net Core not using/requiring these concepts anymore? What was the scenario that required them in the first place?)